packages/labs/testing/README.md

```diff
@@ -31,7 +31,7 @@ Add the plugin to your config file.
 
 ```js
 // web-test-runner.config.js
-import {litSsrPlugin} from '@lit-labs/testing';
+import {litSsrPlugin} from '@lit-labs/testing/web-test-runner.js';
```

Comment:
I wonder if this should be more specific a module? Like `@lit-labs/testing/web-test-runner-ssr-plugin.js`? If we ever had another web test runner plugin, it might be nice to keep the module graph distinct.

Reply:
Hmm the name is kind of getting long but I see your point about wanted to keep it distinct. `@lit-labs/testing/wtr-ssr-plugin.js`? I don't know if `wtr` is a common enough acronym that's as recognizable.

Reply:
Either is OK with me.

Reply:
Went with the longer more descriptive name.